src/aichat/chatapp/data/jokes.csv

```diff
@@ -1,8 +1,8 @@
-give|tell me a joke,"I own the world's worst thesaurus. Not only is it awful, it's awful.",,
-give|tell me a joke,"Hypochondriacs always take things, literally.",,
-give|tell me a joke,"I was going to tell a time traveling joke, but you didn't like it.",,
-give|tell me a joke,"If your parachute doesn't deploy, you have the rest of your life to fix it.",,
-give|tell me a joke,"I just invented a new word: plagarism.",,
-give|tell me a joke,"Rest in peace, boiling water. You will be mist.",,
-give|tell me a joke,"I wonder if the guy who came up with the term 'One Hit Wonder' came up with any other phrases.",,
-give|tell me a joke,"With the rise of self-driving vehicles, it’s only a matter of time before we get a country song where a guy’s truck leaves him too.",,
+give|tell me a joke,"I own the world's worst thesaurus. Not only is it awful, it's awful."

```

Comment:
those trailing commas are intentional,we should leave them in and make sure all our dialog parsing functions can deal with them.

Reply:
The quotes should definitely be deleted, so that's a good catch.